build/repo.props

```diff
@@ -57,13 +57,15 @@
                        $(RepositoryRoot)src\Tools\dotnet-watch\test\TestProjects\**\*.csproj
                        " />
 
+    <ProjectToBuild Include="@(NativeProjects)" Condition="'$(OS)' == 'Windows_NT'"/>
```

Comment:
Is NativeProjects defined somewhere? I don't see \*.vcxproj's getting picked up anywhere.

Reply:
Project references in M.A.Server.IIS, so no need for them.